qiita_db/analysis.py

```diff
@@ -266,6 +275,21 @@ def name(self):
         return conn_handler.execute_fetchone(sql, (self._id, ))[0]
 
     @property
+    def portal(self):
```

Comment:
Changed my mind about this. We don't want to use this across the code base and it is only intended for debugging. So this is a "DO NOT USE" in the codebase, I suggest changing it's name to use a double leading underscore to identify that is super-private.